stories/auth.js

```diff
@@ -25,33 +25,33 @@ class Auth {
                     this.accessToken = this.ssoAuth.getIdToken();
                     this.reportIdentity();
```

Comment:
This seemed to be crashing when I was trying it out - seems like there is no such method. Things started working through when I removed all the mention of `reportIdentity` in this file

Reply:
Ah, thanks! I removed this method as it was "not needed" by storybook but forgot to clear.